examples/grpc_example/BUILD.bazel

```diff
@@ -13,10 +13,15 @@ tidy(
 # MARK: - Gazelle
 
 # Ignore the Swift build folder
-# gazelle:exclude .build
+# gazelle:proto package
 # gazelle:swift_module_naming_convention pascal_case
 # gazelle:swift_generate_proto_libraries true
-# gazelle:swift_generate_grpc_libraries_with_flavors client,client_stubs,server
+# gazelle:swift_proto_compiler swift_proto=//compilers:swift_proto
+# gazelle:swift_proto_compiler swift_client_proto=//compilers:swift_client_proto
+# gazelle:swift_proto_compiler swift_server_proto=//compilers:swift_server_proto
+# gazelle:swift_generate_grpc_libraries_with_flavors swift_client_proto,swift_server_proto
+# gazelle:exclude .build
+# gazelle:exclude bazel-*
```

Comment:
I thought Gazelle was smart enough not to traverse `bazel-*` directories. I have never needed to add this to other projects. Can you explain what prompted this?

Reply:
Ah you're right -- I personally set `build --symlink_prefix=.bazel/` in my bazelrc so all of the bazel-* folders are nested under ".bazel" and hidden from finder. Gazelle doesn't handle this correctly, so I have to exclude the .bazel directory. I forgot it already excluded the normal bazel- paths.